packages/client/integration/step-definitions/evaluation.spec.ts

```diff
@@ -22,6 +23,16 @@ const givenAnOpenfeatureClientIsRegisteredWithCacheDisabled = (
 };
 
 defineFeature(feature, (test) => {
+  beforeAll((done) => {
+    client.addHandler(ProviderEvents.Ready, () => {
+      done();
+    })
+  });
+
+  afterAll(() => {
+    OpenFeature.close();
```

Comment:
You may see an error like: 

```
Jest did not exit one second after the test run has completed.

'This usually means that there are asynchronous operations that weren't stopped in your tests. Consider running Jest with `--detectOpenHandles` to troubleshoot this issue.
```

This is actually a bug in the flagd web provider which I will fix (and add you to). We aren't properly shutting down the event stream. Once this fix is published that will stop.